### PR TITLE
optimization

### PR DESCRIPTION
**Pull request by KannanApp**
React Render and Optimizations

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
     "@testing-library/user-event": "^13.5.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-native-axios": "^0.17.1",
```
Comment (shailendrakanherkar18) [inline package.json]: we need plain axios

File: src/Dashboard/AddTodoForm.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { memo, useState } from "react";
+
+const AddTodoFormComponent = ({addTodo}) => {
+    const [addTodoInput, setAddTodoInput] = useState("");
+    return ( 
+        <div>
+            <input 
+            type="text"
```
Comment (shailendrakanherkar18) [inline src/Dashboard/AddTodoForm.jsx:8]: indentation ? are you using prettier ?

File: package.json (hunk 1 above)
Comment (KannanApp) [reply package.json]: removed

File: src/Dashboard/AddTodoForm.jsx (hunk 2 above)
Comment (KannanApp) [reply src/Dashboard/AddTodoForm.jsx:8]: For this one, testing purpose i added.
Actuall goal is - Add a filter bar with three controls - search, sort and status

**Review by KannanApp — COMMENTED** [review]
removed

**Review by shailendrakanherkar18 — APPROVED** [review]

**Pull request merged**